### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by josephchaouistannard** [noDRM/DeDRM_tools#189] [General]
Hi everyone, 

I'm not a programmer or anything, I just made an account for github so I could say thanks to all the people that make these great tools! I just bought a technical book about animal nutrition and it could only be found with DRM limiting how and where I can read it. So thanks for helping me get the most out of my new book!

**Comment by nifgam** [comment]
Legendary. I can now view my books on a large screen where I can see the detailed charts so much better than on my little Kobo. It also means that I will be buying a bunch more books that I wouldn't have done if not for this software, which is therefore supporting the publishers and authors.
